task-native/src/main/java/org/ballerinalang/stdlib/task/utils/TaskCallBack.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.stdlib.task.utils;
+
+import io.ballerina.runtime.api.async.Callback;
+import io.ballerina.runtime.api.values.BError;
+import org.quartz.JobExecutionContext;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.PrintStream;
+
+/**
+ * The class handles the output of job execution by the trigger.
+ *
+ * @since 1.2.0
+ */
+public class TaskCallBack implements Callback {
+
+    JobExecutionContext jobExecutionContext;
+    private static final Logger LOG = LoggerFactory.getLogger(TaskCallBack.class);
+
+    public TaskCallBack(JobExecutionContext jobExecutionContext) {
+        this.jobExecutionContext = jobExecutionContext;
+    }
+
+    @Override
+    public void notifySuccess(Object o) {
+    }
+
+    @Override
+    public void notifyFailure(BError bError) {
+        PrintStream print = System.out;
```

Comment:
Let's print as system err. Check whether we can use log API and log similar to the log message format